src/lib/core/components/computations/ComputationStepContainer.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useUITheme } from '@veupathdb/coreui/dist/components/theming';
+import {
+  NumberedHeader,
+  NumberedHeaderProps,
+} from '../../../workspace/Subsetting/SubsetDownloadModal';
```

Comment:
maybe we should move `NumberedHeader` to CoreUI someday?

Reply:
I created an issue for this https://github.com/VEuPathDB/CoreUI/issues/137